app/src/main/java/com/willowtree/vocable/settings/EditCategoryOptionsFragment.kt

```diff
@@ -46,7 +46,7 @@ class EditCategoryOptionsFragment : BaseFragment<FragmentEditCategoryOptionsBind
             findNavController().popBackStack()
         }
 
-        binding.removeCategoryButton.action = {
+        binding.removeCategoryButton.setOnClickListener { 
```

Comment:
We should use `action` here instead of setting a click listener. The `action` is what will get triggered by head tracking and it will get automatically triggered by a user clicking the screen.

Reply:
Gotcha - I switched this view from a ConstraintLayout to a VocableImageButton and it didn't like the action keyword.  Could have just been an IDE issue, I'll look into it